client/components/search/index.jsx

```diff
@@ -38,7 +38,8 @@ module.exports = React.createClass( {
 		onKeyDown: React.PropTypes.func,
 		disableAutocorrect: React.PropTypes.bool,
 		onBlur: React.PropTypes.func,
-		searching: React.PropTypes.bool
+		searching: React.PropTypes.bool,
+		forceCloseButton: React.PropTypes.bool
```

Comment:
@bluefuton my argument is that if search is not pinned, the X shouldn't show if the input is empty, because it does nothing. If search is pinned, the X collapses the search, so it has a function.